packages/compass-saved-aggregations-queries/src/stores/aggregations-queries-items.ts

```diff
@@ -1,5 +1,102 @@
-function reducer(state = {}): typeof state {
+import { remote } from 'electron';
+import { join } from 'path';
+import fs from 'fs';
+import { AnyAction } from 'redux';
+import { FavoriteQueryCollection } from '@mongodb-js/compass-query-history';
+
+import { actions } from '../actions/aggregations-queries-actions';
+interface Item {
+  id: string;
+  name: string;
+  namespace: string;
+  lastModified: number;
+}
```

Comment:
I think it will get clear going on with the implementation, but this structure probably is not going to be enough. Probably you'll need some kind of `type: 'aggregation' | 'query'` field here.

Having a common "Item" type is very nice. Probably we don't want Query and Aggregation objects to share the same "space" for keys, they are unrelated objects, and technically keys with the same name but different meanings may clash.

I'd suggest something like this:

```
interface AggregationItem extend Item {
   type: 'aggregation',  value: Aggregation
}
```

```
interface FindQueryItem extend Item {
   type: 'findQuery',  value: FindQuery
}
```